src/components/map-page-components/mapLayout/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Layout } from 'antd';
+
+const mapLayoutHeight = 'calc(100vh - 83px)';
+
+const MapLayout: React.FC = ({ children }) => {
+  return (
+    <>
+      <Layout style={{ height: mapLayoutHeight }}>{children}</Layout>
```

Comment:
Is there a reason we're setting this inline as opposed to using styled-components?